core/src/com/unciv/ui/utils/Sounds.kt

```diff
@@ -2,22 +2,40 @@ package com.unciv.ui.utils
 
 import com.badlogic.gdx.Gdx
 import com.badlogic.gdx.audio.Sound
+import com.badlogic.gdx.files.FileHandle
 import com.unciv.UncivGame
 import com.unciv.models.UncivSound
 
 object Sounds {
-    private val soundMap = HashMap<UncivSound, Sound>()
+    private val soundMap = HashMap<String, Sound?>()
 
-    fun get(sound: UncivSound): Sound {
-        if (!soundMap.containsKey(sound)) {
-            soundMap[sound] = Gdx.audio.newSound(Gdx.files.internal("sounds/${sound.value}.mp3"))
+    fun get(sound: UncivSound): Sound? {
+        val fileName = if (sound == UncivSound.Custom) sound.custom!! else sound.value
+        if (!soundMap.containsKey(fileName)) {
+            val mods = mutableSetOf<String>()
+            if (UncivGame.Current.isInitialized) {
+                mods.addAll(UncivGame.Current.settings.visualMods)
```

Comment:
All things considered, it seems to me that A. the soundMap change is correct, but B. we should make a get(soundName:String) function, and have get(sound:UncivSound) call it. When we want a custom sound, we call get(soundName:String).